core/src/main/java/com/redhat/lightblue/client/ExpressionPart.java

```diff
@@ -0,0 +1,26 @@
+package com.redhat.lightblue.client;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+
+/**
+ * Base class for any expression part that is backed by a Json node
+ */
+public abstract class ExpressionPart implements JsonObj {
+    protected JsonNode node;
```

Comment:
It should be, but just because of Literal(Object x), it can't be.

On Thu, Aug 6, 2015 at 12:03 PM, Dennis Crissman <notifications@github.com>
wrote:

> In core/src/main/java/com/redhat/lightblue/client/ExpressionPart.java
> <https://github.com/lightblue-platform/lightblue-client/pull/144#discussion_r36445321>
> :
>
> > @@ -0,0 +1,26 @@
> > +package com.redhat.lightblue.client;
> > +
> > +import com.fasterxml.jackson.databind.JsonNode;
> > +import com.fasterxml.jackson.databind.node.ObjectNode;
> > +import com.fasterxml.jackson.databind.node.JsonNodeFactory;
> > +
> > +/**
> > + * Base class for any expression part that is backed by a Json node
> > + */
> > +public abstract class ExpressionPart implements JsonObj {
> > +    protected JsonNode node;
>
> Should node be final?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lightblue-platform/lightblue-client/pull/144/files#r36445321>
> .
>